TFDS_Python_Interoperability.ipynb

```diff
@@ -0,0 +1,310 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "TFDS Python Interoperability.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "swift",
+      "display_name": "Swift"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "aNAkr1ldsRaR",
+        "colab_type": "text"
+      },
+      "source": [
+        "# TFDS Python Interoperability with Swift\n",
+        "\n",
+        "S5TF Team is actively developing a new `Datasets` library, however in the meantime we can take advantage of the excellent Python interoperability to load datasets from [TFDS](https://github.com/tensorflow/datasets/). Below is an example of how to do this with the MNIST dataset."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "udEcj1K4oWMY",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Importing dependencies"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "kZRlD4utdPuX",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import TensorFlow\n",
+        "import Python\n",
+        "\n",
+        "let plt = Python.import(\"matplotlib.pyplot\")\n",
+        "let np = Python.import(\"numpy\")\n",
+        "let tfds = Python.import(\"tensorflow_datasets\")\n",
+        "let tf = Python.import(\"tensorflow\")\n",
+        "\n",
+        "%include \"EnableIPythonDisplay.swift\"\n",
+        "IPythonDisplay.shell.enable_matplotlib(\"inline\")"
```

Comment:
These lines should be in their own cell because of [this](https://bugs.swift.org/browse/TF-183). See [this guide](https://www.tensorflow.org/swift/tutorials/python_interoperability#displaying_images).